### PR TITLE
Yorkies

### PR DESCRIPTION
**Pull request by standardtoaster**
Updated yorkie recipe. Looking for feedback on
* Scaling
* Ingredient formatting
* cooking log

### CODE REVIEW

**Comment by standardtoaster** [comment]
Also, I guess I should squash these in the future?

File: yorkies.md [hunk 1]
```diff
@@ -18,23 +18,41 @@ The BBC has more [good references](https://www.bbcgoodfood.com/howto/guide/top-t
...
-* 140 grams cups all-purpose flour
-* 1/4 teaspoon salt
-* Lard or Duck Fat or oil
+> **_NOTE:_** This is designed to be scaled using eggs. 1 egg is ~ 1.25 yorkies, so 5 eggs gets you about 6 muffin tin sized yorkies
+
+> **_NOTE:_** Flour ratio is WIP. I've seen anywhere from ~28-35 grams per egg. 28 was super "eggy", 35 was more like a popover 
+
+### Batch of 6
+
+| Measure       | Ounces | Grams | Ingredients             | %    |
```
Comment (dgadling) [inline yorkies.md]: Get rid of the `Ounces` column since you never use it.

File: yorkies.md [hunk 2]
```diff
@@ -18,23 +18,41 @@ The BBC has more [good references](https://www.bbcgoodfood.com/howto/guide/top-t
...
+| ------------- | ------ | :---- | ----------------------- | ---- |
```
Comment (dgadling) [inline yorkies.md]: Grams columns should be **right** aligned if anything, right?

**Review by dgadling — CHANGES_REQUESTED** [review]
Please fix

File: yorkies.md [hunk 3]
```diff
@@ -67,4 +67,12 @@ The BBC has more [good references](https://www.bbcgoodfood.com/howto/guide/top-t
...
 5. Fill each tin to 3/4 full
-6. Bake 15-20 minutes until golden brown
\ No newline at end of file
+6. Bake 15-20 minutes until golden brown
+
+
+
+# Cooking Log
+
+**x-05-20:** Attempted "breakfast" yorkies with jam. One came out nicely, the rest leaked into the tin. Wasn't fun to clean.
```
Comment (dgadling) [inline yorkies.md]: ISO 8601 exists for a reason! YYYY-MM-DD plz ; `?`'s if you're not sure.

**Review by dgadling — CHANGES_REQUESTED** [review]
See inline comments.

File: yorkies.md (hunk 3 above)
Comment (standardtoaster) [reply yorkies.md]: UK Time is better, but fineeeeeeee

**Review by dgadling — APPROVED** [review]
lgtmacro

**Pull request merged**